aikau/src/main/resources/alfresco/services/DialogService.js

```diff
@@ -474,8 +474,8 @@ define(["dojo/_base/declare",
             duration: payload.duration || 0,
             fullScreenMode: payload.fullScreenMode || false,
             fullScreenPadding: !isNaN(payload.fullScreenPadding) ? payload.fullScreenPadding : 10,
-            widgetsContent: payload.widgetsContent,
-            widgetsButtons: payload.widgetsButtons,
+            widgetsContent: lang.clone(payload.widgetsContent),
+            widgetsButtons: lang.clone(payload.widgetsButtons),
```

Comment:
This doesn't appear to cause a problem, in the unit test page we have widgetsContent but not widgetsButtons and that doesn't result in any errors